src/client.ts

```diff
@@ -87,6 +99,35 @@ export class APIClient {
       body,
     });
 
+    // According to BambooHR documentation, a 429 error is a limit exceeded
+    // regarding adding employees.  A 503 is a "currently unavailable" 
+    // error that is most commonly due to rate limiting.  It may contain
+    // a 'Retry-After' value in the header specifying how long to wait.
+    // Unfortunately, we don't have documenation or examples for this, and
```

Comment:
nit: documentation

Reply:
Went ahead and updated my inline comment and added information in documentaiton.md after I saw this. :)